woocommerce-services.php

```diff
@@ -1121,21 +1095,6 @@ public function rest_api_init() {
 			add_filter( 'rest_request_before_callbacks', array( $this, 'log_rest_api_errors' ), 10, 3 );
 		}
 
-		/**
-		 * If the required v3 REST API endpoints haven't been loaded at this point, load the local copies of said endpoints.
-		 * Delete this when the "v3" REST API is included in all the WC versions we support.
-		 */
-		public function wc_api_dev_init() {
```

Comment:
Removed instead of loading. See: https://github.com/Automattic/woocommerce-services/pull/2812/files#r1753874334